app/controllers/usersController.js

```diff
@@ -2,7 +2,37 @@
 
 const errors = require('../errors'),
   logger = require('../logger'),
-  User = require('../models').User;
+  User = require('../models').User,
+  bcrypt = require('bcrypt'),
+  userService = require('../services/userService'),
+  sessionManager = require('../services/sessionManager');
+
+exports.login = (req, res, next) => {
+  const userLogin = req.body
+    ? {
+        email: req.body.email,
+        password: req.body.password
+      }
+    : {};
+
+  userService.getByEmail(userLogin.email).then(u => {
```

Comment:
Please try using declarative names for variables, instead of `u` you can call it `user` and it makes easier for someone else read it later.